static/js/util/util.js

```diff
@@ -455,3 +455,30 @@ export function getUserDisplayName(profile: Profile): string {
 
   return `${first} ${last}${preferred_name}`;
 }
+
+function intersperse(arr, elementFunc) {
+  let resultArray = [], i=0;
+  if (i < arr.length)
+    resultArray.push(arr[i++]);
+  while (i < arr.length)
+    resultArray.push(elementFunc(i), arr[i++]);
+  return resultArray;
+}
+
+/**
+ * Takes a list of components and renders them with separator components in between each one
+ */
+export function renderSeparatedComponents(
+  components: Array<React$Element<*>|null>,
+  separator: string
+): Array<React$Element<*>|null> {
+  return intersperse(components, (i) => ( <span key={`separator-${i}`}>{ separator }</span> ));
+}
+
+/**
+ * Returns the value of an object's property by name, or a default if (a) the property name
+ * is null/undefined or (b) the property doesn't exist in the object.
+ */
+export function getPropertyOrDefault(obj: Object, property: ?string, defaultValue: any) {
+  return _.get(obj, property, defaultValue);
+}
```

Comment:
Hm, well we do use `_.get` in a bunch of places with a third argument already, so I'm not so sure this would give us a big win in terms of swapping out lodash functionality for lodash. I'm also not so opinionated on this, so if you wanna keep it go for it.